install_config/router/f5_router.adoc

```diff
@@ -243,27 +241,27 @@ BIG-IP® documentation] for more information.
 . xref:f5-configuring-the-virtual-servers[Configure a virtual server] for the
 custom partition.
 
-. Deploy the F5 router using the `--external-host-partition-path` flag to specify a partition path:
+. Deploy the F5 router plug-in using the `--external-host-partition-path` flag to specify a partition path:
```

Comment:
To specify a partition path, deploy the F5 router plug-in using the `--external-host-partition-path` flag: